fastlane/Fastfile

```diff
@@ -696,7 +703,7 @@ REPOSITORY_NAME = 'WordPress-Android'
       "release_note_#{values[0]}#{values[1]}" => { desc: "changelogs/#{options[:build_number]}.txt", max_size: 500, alternate_key: "release_note_short_#{values[0]}#{values[1]}" },
       play_store_app_title: { desc: 'title.txt', max_size: 30 },
       play_store_promo: { desc: 'short_description.txt', max_size: 80 },
-      play_store_desc: { desc: 'full_description.txt', max_size: 0 }
+      play_store_desc: { desc: 'full_description.txt', max_size: 4000 }
```

Comment:
This lack of a limit passed in our tooling in the past explains why we had `full_description.txt` translations that were way larger than the 4000 characters limit present in the repo until now.

Now that I've automated the metadata upload by the CI on each production build (see L579 above), I had to make sure that we discarded any translation for those larger than the limit, to avoid the upload and metadata update in PlayStore to fail (see p1648830761024449-slack-C5ALGJYEP).

This change is what led to the (appropriate) deletion of many `full_description.txt` files in this PR (basically all the ones for which the translation exceeded that limit and should never have been pulled and stored in the repo in the first place since they never made it to the PlayStore in reality)